pinot-core/src/main/java/com/linkedin/pinot/core/io/writer/impl/v1/FixedByteChunkSingleValueWriter.java

```diff
@@ -47,24 +47,24 @@
 @NotThreadSafe
 public class FixedByteChunkSingleValueWriter extends BaseChunkSingleValueWriter {
 
-  private static final int VERSION = 1;
+  private static final int VERSION = 2;
```

Comment:
suggest rename VERSION to CURRENT_VERSION

Reply:
Done